[MTM-64175] adding information of different content type for certific… (#3556)
SoftwareAG/c8y-docs: content/device-certificate-authentication/device-enroll-and-re-enroll.md

* If tenant's keypair is not found then an error occurred with message `Failed to retrieve tenant keypair`.
* If the request does not contain a valid [CertificateSigningRequest](https://en.wikipedia.org/wiki/Certificate_signing_request) an error will be returned.

{{< c8y-admon-info >}}

> As per [EST standards](https://datatracker.ietf.org/doc/html/rfc7030#autoid-58), the certificate in response will be in `PKCS7` format by default. Client can optionally request for a `PKCS10` format by sending `Accept: application/pkcs10` in the request header.

Reviewer: ```suggestion
As per [EST standards](https://datatracker.ietf.org/doc/html/rfc7030#autoid-58), the certificate in response is in `PKCS7` format by default. Clients can optionally request for a `PKCS10` format by sending `Accept: application/pkcs10` in the request header.
```

    HTTP/1.1 201 OK
    Content-Type: application/pkcs7-mime;smime-type=certs-only
    Content-Length: ...
    {
        MIIBpQYJKoZIhvcNAQcCoIIBljCCAZICAQExADALBgkqhkiG9w0BBwGgggF6MIIB
        djCCARygAwIBAgIDCnqWMAoGCCqGSM49BAMCMBcxFTATBgNVBAMTDGVzdEV4YW1w
        bGVDQTAeFw0yNTA1MjIwNjUwMzJaFw0yNjA1MjIwNjUwMzJaMBQxEjAQBgNVBAMM
        CWV4YW1wbGUwMTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABI882KOUAoc5DCKF
        u0zansWbtL84zKJa7SnconIF3AYjvHl3zBUnkv+/7+tSA13mBbYdq4oGh6Yg83zj
        zhEJ0eujWjBYMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgeAMB0GA1UdDgQWBBTsMXNm
        ZMWaR8B74Imk6XckmUvOXTAfBgNVHSMEGDAWgBQa3zmEwlbmbM8qtCal/QzSQ/U9
        PjAKBggqhkjOPQQDAgNIADBFAiEAhrhYYDGcvsFkJpg9U4fr8d7LuqearEnMn+TF
        RWWgCmICIGPUvHHKzSc8GbgwuKddhe0Ef+HTamyEmqQBX3z/iE3GMQA=
    }

> and an example of a REST request with `Accept: application/pkcs10` (optional):    

Reviewer: ```suggestion
And an example of a REST request with `Accept: application/pkcs10` (optional):    
```
